Formula/rtags.rb

```diff
@@ -28,6 +28,45 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
+    The instance of rtags that run via brew services listens to
+    the socket file at #{var}/.rdm
+
+      rc --socket-file=#{var}/.rdm ...
+    EOS
+  end
+
+  plist_options :startup => false
```

Comment:
`false` doesn't need stating. Usually if not `plist_options :startup` you'd set something like:
```ruby
plist_options :manual => "rdm --log-file=#{var}/log/rtags.log --socket-file=#{var}/.rdm"
```
Or such.